ipi_tests/regression_tests/tests/NEB/FIRE/zundel/files_to_check.txt

```diff
@@ -1,6 +1,6 @@
         filename                        format      icol,rtol,atol  icol,rtol,atol ...
 --------------------------------------------------------------------------------------
-ref_simulation.neb                      numpy       0,0.0,0.0  6,1e-7,1e-6  7,1e-7,1e-6  8,1e-7,1e-6  9,1e-7,1e-6  10,1e-7,1e-6  11,1e-7,1e-6
+ref_simulation.neb                      numpy       all,1.e-7,1e-6
```

Comment:
I would also use the all syntax for the phonon case. There there are several checks on dynmat-like files with a load of columns, and I had only selected a random few. It's a great use case for `all`